src/app/modules/file-upload/file-upload.component.ts

```diff
@@ -20,6 +20,11 @@ export class FileUploadComponent implements OnInit {
   }
 
   uploadFile(event) {
+    if (event.files) {
+      // ts workaround since `files` doesn't exist on EventTarget
+      // and I can't cast types in the template
+      event = event.files
+    }
```

Comment:
Do we need the if statement?  You might be able to get rid of it and just loop through event.files or event?.files if there's a case where files doesn't exist.

Reply:
That's fancy! 😄 
Yes, that works. And really what you already have works too...I just didn't realize that event could be an array when I made the comment.

Reply:
I feel that, it doesn't really make a lot of sense 🥴 

Let me also rename this variable to something that makes sense like `filesToUpload` so that we're not reassigning a variable called `event` to be a bunch of files 🙃